wgpu/src/lib.rs

```diff
@@ -1920,12 +1948,6 @@ impl Instance {
     /// If the specified display and window handle are not supported by any of the backends, then the surface
     /// will not be supported by any adapters.
     ///
-    /// # Safety
```

Comment:
We really need to add documentation on how this is used and how what you pass in relates to the output lifetime of the surface.

While the signature and trait definition does technically have the information, this isn't readily grokable, especially to rust newcomers.